content/swagger/resources/product_models/routes/product_models_code.yaml

```diff
@@ -7,7 +7,7 @@ get:
     - "2.0"
     - "2.1"
     - "2.2"
-  description: This endpoint allows you to get the information about a given product model.
+  description: This endpoint allows you to get the information about a given product model. In the Entreprise Edition, since the v2.3, permissions based on your user groups are applied to the product model you request.
```

Comment:
It's from the version 2.0 no? What's new about that in 2.3? Because we are filtering data.

Reply:
It was not officially working in 2.0, 2.1 and 2.2. And the proposals were wrongly created.

Reply:
My bad, you are talking about the get. You are right, it's working since 2.0. Copy-paste evil.